myhpi/core/templates/core/minutes_list.html

```diff
@@ -1,86 +0,0 @@
-{% extends "base.html" %}
-{% load wagtailcore_tags %}
-{% load i18n %}
```

Comment:
What has happened here? Diff is showing this file as deleted?

Reply:
(or at least all lines deleted)

Reply:
yep djhtml crashed when processing this file because of the hardcoded emojis. fixed now